source/common/http/codec_helper.h

```diff
@@ -7,6 +7,18 @@ namespace Http {
 
 class StreamCallbackHelper {
 public:
+  void runLowWatermarkCallbacks() {
+    for (StreamCallbacks* callbacks : callbacks_) {
+      callbacks->onBelowWriteBufferLowWatermark();
```

Comment:
I believe Http2CodecImplFlowControlTest.EarlyResetRestoresWindow validates this works and the ASSERT on destruct of the codec should hopefully futureproof it.